ansible_base/resource_registry/resource_server.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta
+from typing import TypedDict
+
+import jwt
+from django.conf import settings
+
+from ansible_base.resource_registry.models import service_id
+
+
+class ResourceServerConfig(TypedDict):
+    URL: str
+    SECRET_KEY: str
+    VALIDATE_HTTPS: bool
+    JWT_ALGORITHM: str
+
+
+def get_resource_server_config() -> ResourceServerConfig:
+    defaults = {"JWT_ALGORITHM": "HS256", "VALIDATE_HTTPS": True}
```

Comment:
I've been thinking about things like the algorithms and wondering if we should variablize that incase someone wanted to run with HS512 or whatever else. Just a thought, not asking this PR to change.

Reply:
It is variablized. You can pick HS256 -> HS512 on the server and then configure that here.